lib/node_modules/@stdlib/math/base/special/sincos/docs/repl.txt

```diff
@@ -0,0 +1,37 @@
+
+{{alias}}( [out,] x )
+    Simultaneously computes the sine and cosine of a number.
+
+    Parameters
+    ----------
+    [out]: array
```

Comment:
We don't use square brackets in the REPL help texts, but a `(optional)` after the type instead. This should be `out: Array|TypedArray|Object (optional)`, since one can supply any collection for `out`.